includes/core/server/request/class-refund-charge.php

```diff
@@ -25,6 +25,13 @@ class Refund_Charge extends Request {
 
 	const REQUIRED_PARAMS = [ 'charge' ];
 
+	/**
+	 * Used to set WordPress filter hook that will be executed when send() function is called.
```

Comment:
I see that "filter hook" is used thoughout all request classes. Let's mass-replace with. Also, there is a "the" missing before "WordPress".

Reply:
Addressed here: https://github.com/Automattic/woocommerce-payments/pull/7314/commits/e39ac1fe049de9ff790af034762fead702a8fbe9